crates/goose/src/agents/recipe_tools/dynamic_task_tools.rs

```diff
@@ -81,9 +81,6 @@ pub struct TaskParameter {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub retry: Option<JsonObject>,
 
-    #[serde(skip_serializing_if = "Option::is_none")]
-    pub context: Option<Vec<String>>,
-
     #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
so yeah, this entire struct, why can't it be a recipe? that way the tool would have no choice but provide us with a valid recipe, rather than Vec<JsonObject>s that might or might not fit what we expect?

that would solve: https://github.com/block/goose/issues/5241